app/util/links.ts

```diff
@@ -19,6 +19,8 @@ export const links = {
   imagesDocs: 'https://docs.oxide.computer/guides/creating-and-sharing-images',
   preparingImagesDocs:
     'https://docs.oxide.computer/guides/creating-and-sharing-images#_preparing_images_for_import',
+  identityProvidersDocs:
+    'https://docs.oxide.computer/guides/system/completing-rack-config#_configure_silo_identity_provider',
```

Comment:
Not suggesting a change here, just musing — the linked table is pretty good. It's a shame the generated doc for the endpoint is not nearly as helpful. It's kind of hard to see how it could be as good, since the table includes hand-written notes about, e.g., how the name will be used in context. I wonder if there's a way we could add our own notes in the API structs in addition to the default `Name` docs. Unfortunately these comes from another struct:

```rs
#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
pub struct SamlIdentityProviderCreate {
    #[serde(flatten)]
    pub identity: IdentityMetadataCreateParams,
```

<img width="861" alt="image" src="https://github.com/user-attachments/assets/77960565-0422-4880-9fdc-e0a79c43fc6a">

<img width="696" alt="image" src="https://github.com/user-attachments/assets/df0d1460-5687-4cb2-adf0-8c2b641d9aaa">